index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter = 1
+counter = 2
+
+console.log(counter)
```

Comment:
This works, but it's best to get into the habit of using a variable declaration keyword such as `let` for this example